### PR TITLE
Add Home page Features

### PR DESCRIPTION
**Pull request by BenMKT**
# App Implementations: 

- Created a homepage dynamically using ES6 modules for best practices.
- Utilized TV API.

### CODE REVIEW

File: src/Modules/tvApi.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+const myUrl = 'https://api.tvmaze.com/shows/';
+
+const container = document.getElementById('cardContainer');
+
+// const display = (json) => {
+//   document.body.appendChild(myImg)
+// }
```
Comment (IsmaelMastronardi) [inline src/Modules/tvApi.js]: Please delete this commented-out code. The project will look better and more professional without unnecessary code on it.

File: src/Modules/tvApi.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  // Append the title and the imgContainer to the card
+  imgContainer.appendChild(myImg);
+  card.appendChild(title);
+  card.appendChild(imgContainer);
+  card.appendChild(likeBtn);
+  card.appendChild(commentBtn);
+  container.appendChild(card);
+
+  // Return the card element
+  //   return card;
```
Comment (IsmaelMastronardi) [inline src/Modules/tvApi.js]: Please also remove this commented code for the same reason as before.

File: src/Modules/tvApi.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+// },
+// {
+//   "name": "show 3",
+//   "id": "3"
+// }
+// ]
+
+// myArr.forEach((elem)=>{
+//   showImg(elem)
+// })
```
Comment (IsmaelMastronardi) [inline src/Modules/tvApi.js]: Please also delete this commented code, for the same reason as the other comments.

File: src/index.html [hunk 4]
```diff
@@ -1,11 +1,66 @@
...
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div>
+        <div>
+          <img src="" alt="" />
+          <h2></h2>
+          <button type="button"><img src="" alt="love icon" />likes</button>
+          <button type="button">Comments</button>
+        </div> -->
```
Comment (IsmaelMastronardi) [inline src/index.html]: Please also delete these commented divs.

File: src/index.html [hunk 5]
```diff
@@ -1,11 +1,66 @@
...
+      </section>
+    </main>
+    <footer>
+      <p>Created by Microverse under CC license</p>
+    </footer>
+    <!-- <script type="module" src="index.js"></script> -->
```
Comment (IsmaelMastronardi) [inline src/index.html]: Please also delete this commented script line.

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,20 @@
-body {
+/* body {
   background-color: red;
+} */
```
Comment (IsmaelMastronardi) [inline src/style.css]: Please delete this commented code.

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,20 @@
...
+
+.img-container {
+  height: 200px;
+  overflow: hidden;
+}
+
+img {
+  width: 100%;
+  object-fit: cover;
 }
```
Comment (IsmaelMastronardi) [inline src/style.css:19]: The project will look even better with some more styles added to it.

**Review by IsmaelMastronardi — COMMENTED** [review]
Hello Benson, The home page is looking great 🔥, But i have a few suggestions for you.

File: src/Modules/tvApi.js [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+// }
+
+const createCard = (obj, json) => {
+  // Create a div element with class card
+  const card = document.createElement('div');
+  card.className = 'card';
+
+  // Create a h3 element with the name of the show
+  const title = document.createElement('h3');
+  title.textContent = obj.name;
```
Comment (IsmaelMastronardi) [inline src/Modules/tvApi.js]: The title should use the name given in the JSON, not the object, we are not supposed to use hard-coded titles.

File: src/style.css (hunk 6 above)
Comment (BenMKT) [reply src/style.css]: - Deleted commented code

File: src/index.html (hunk 5 above)
Comment (BenMKT) [reply src/index.html]: - Deleted commented code

File: src/index.html (hunk 4 above)
Comment (BenMKT) [reply src/index.html]: - Deleted commented code

File: src/style.css (hunk 7 above)
Comment (BenMKT) [reply src/style.css:19]: - Will add more styles after implementing all app functionality

File: src/Modules/tvApi.js (hunk 3 above)
Comment (BenMKT) [reply src/Modules/tvApi.js]: - Deleted commented code

File: src/Modules/tvApi.js (hunk 2 above)
Comment (BenMKT) [reply src/Modules/tvApi.js]: - Deleted commented code

File: src/Modules/tvApi.js (hunk 1 above)
Comment (BenMKT) [reply src/Modules/tvApi.js]: - Deleted commented code

File: src/Modules/tvApi.js (hunk 8 above)
Comment (BenMKT) [reply src/Modules/tvApi.js]: - Refactored to use the name given in the JSON.

File: src/Modules/tvApi.js (hunk 1 above)
Comment (IsmaelMastronardi) [reply src/Modules/tvApi.js]: Great, the branch is ready to be merged. Congratulations.

**Pull request merged**